src/spacel/provision/app/ingress_resource.py

```diff
@@ -130,3 +146,14 @@ def _app_eips(self, orbit_region, app):
             if 'does not exist' in e_message:
                 return eips
             raise e
+
+    @staticmethod
+    def _is_rfc1918(ip_match):
+        first_octet = ip_match.group(1)
+        if first_octet == '10':
```

Comment:
Good catch; though someone inputting `010` is either: 1) an idiot, 2) an attacker.
I'll add a guard for the latter :P

Lazy rebasing/non-critical fix means this change will happen in #71 and I'll just merge this as-is.